extensions/objects/templates/svg-filename-editor.component.ts

```diff
@@ -804,6 +814,9 @@ angular.module('oppia').component('svgFilenameEditor', {
       };
```

Comment:
Is 10 and 0 some default value? If so, assign to a descriptive variable and use those here.

Reply:
Done.